docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,51 +1,125 @@
 
-**OpenTelemetry API/SDK with Elastic APM agents**
+* <<apm-otel-elastic-distros,Elastic Distributions for OpenTelemetry Language SDKs>>
+* <<apm-otel-api-sdk-elastic-agent,OpenTelemetry API/SDK + Elastic APM agent>>
+* <<apm-otel-api-sdk,OpenTelemetry API/SDK>>
+* <<apm-otel-collector,OpenTelemetry Collector>>
+* <<apm-otel-lambda,Lambda collector exporter>>
+
+[float]
+[[apm-otel-elastic-distros]]
+== Elastic Distributions for OpenTelemetry Language SDKs
+
+Elastic offers several distributions for OpenTelemetry language SDKs.
+A _distribution_ is a customized version of an upstream OpenTelemetry repository.
+Each Elastic Distribution for OpenTelemetry is a wrapper around an https://opentelemetry.io/docs/languages/[OpenTelemetry Language SDK].
+
+image::images/apm-otel-distro.png[]
+
+With an Elastic Distribution for OpenTelemetry Language SDK you have access to all the features of the OpenTelemetry SDK that it customizes, plus:
+
+* You may get access to SDK improvements and bug fixes contributed by the Elastic team _before_
+the changes are available upstream in the OpenTelemetry repositories.
+* The distro preconfigures the collection of tracing and metrics signals, applying some opinionated defaults,
+such as which sources are collected by default.
+
+// TODO: Update links when distro docs are markdown-ified
+Get started with an Elastic Distribution for OpenTelemetry Language SDK:
+
+* preview:["The Elastic Distribution for OpenTelemetry Java is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-java[Java]
+* preview:["The Elastic Distribution for OpenTelemetry .NET is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-dotnet[.NET]
+* preview:["The Elastic Distribution for OpenTelemetry Node.js is not yet recommended for production use. Functionality may be changed or removed in future releases. Alpha releases are not subject to the support SLA of official GA features."] https://github.com/elastic/elastic-otel-node[Node.js]
+
+[NOTE]
+====
+For more details about OpenTelemetry distributions in general,
+visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
+====
+
+[float]
+[[apm-otel-api-sdk-elastic-agent]]
+== OpenTelemetry API/SDK + Elastic APM agent
+
+Use the OpenTelemetry API/SDKs with Elastic APM agents to translate OpenTelemetry API calls to Elastic APM API calls.
+This allows you to reuse your existing instrumentation to create Elastic APM transactions and spans--avoiding
+vendor lock-in and having to redo manual instrumentation.
+
+// Do we want to say anything about why you wouldn't choose this
+// method to send data to Elastic?
```

Comment:
The general downside of this approach is that not all features of the OpenTelemetry API are supported. Agent docs have a full list of caveats.